### PR TITLE
Update VIC intro page with new content

### PR DESCRIPTION
**Pull request by jbalboni**
Signed in, not id proofed:
localhost_3001_veteran-id-card_apply_introduction 5

Not signed in:
localhost_3001_veteran-id-card_apply_introduction 3

Signed in and id proofed:
localhost_3001_veteran-id-card_apply_introduction 4

### CODE REVIEW

File: src/js/vic-v2/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -136,16 +117,29 @@ class IntroductionPage extends React.Component {
...
+          </p>
+          <strong>Begin the Veteran ID Card application without signing in.</strong>
+        </div>}
         <SaveInProgressIntro
           buttonOnly
           pageList={this.props.route.pageList}
           startText="Start the VIC Application"
           {...this.props.saveInProgressActions}
           {...this.props.saveInProgress}/>
+        {(!signedIn || !idProofed) && <p><a href="/faq">Get more information about signing in to Vets.gov</a>.</p>}
```
Comment (cehsu) [inline src/js/vic-v2/components/IntroductionPage.jsx]: Should we remove the period at the end here or include it in the link?

File: src/js/vic-v2/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -136,16 +117,29 @@ class IntroductionPage extends React.Component {
...
             <li className="process-step list-five">
               <div><h5>Decision</h5></div>
-              <p>Once we verify your application, you should receive your Veteran ID Card in the mail in about 60 days.</p>
+              <p>Once we verify your application, you should receive your Veteran ID Card by mail in about 60 days.</p>
             </li>
           </ol>
         </div>
+        {signedIn && !idProofed && <div>
+          <p><strong>Verify your identity before you apply for a Veteran ID Card.</strong>
+            <a className="usa-button usa-button-primary" href="/verify?next=%2Fveteran-id-card%2Fapply">Verify Your Identity</a>
```
Comment (cehsu) [inline src/js/vic-v2/components/IntroductionPage.jsx]: Should we have a line break before the apply link?

**Review by peggygannon — COMMENTED** [review]
@jbalboni Here are my edits to the updated page. Do you want me to input these directly into the files? 

signed in/not ID proofed:
1. We're missing the following bullet in the photo requirement list. It should go above that last bullet (Be in a .jpeg, .png, .bmp, or .tiff file format)
--Show what you look like now (a photo taken sometime in the last 10 years), **and**

2. Photo examples: 
please change header to: 
Examples of good photo IDs (to match how this title appears on the photo upload page)

3. bullet #2
This doesn't have some of the updated copy. The below still needs to be updated:
--Proof of your identity (which could be a driver’s license, a passport, or the ability to answer questions based on private and public data like your credit report)

**In addition to providing extra security measures, when you’re signed in to your account, your application process can go more smoothly. Here’s why:**
--You can get your card sooner because we can confirm your Veteran status more quickly.
--We can prefill part of your application based on your account details.
--You can save your form in progress, and come back later to finish filling it out.

4. Buttons
--bump down Verify Your Identity button to its own line
--The last button should be Start the Veteran ID Card Application
as we don't use VIC acronym on this page. 
If we think that Start the Veteran ID Card Application will be too long, then we'll need to incorporate VIC into the copy on this page. 


Not signed in:
1. We're missing the following bullet in the photo requirement list. It should go above that last bullet (Be in a .jpeg, .png, .bmp, or .tiff file format)
--Show what you look like now (a photo taken sometime in the last 10 years), **and**

2. Photo examples: 
please change header to: 
Examples of good photo IDs (to match how this title appears on the photo upload page)

3. bullet #2
This doesn't have some of the updated copy. The below still needs to be updated:
--Proof of your identity (which could be a driver’s license, a passport, or the ability to answer questions based on private and public data like your credit report)

**In addition to providing extra security measures, when you’re signed in to your account, your application process can go more smoothly. Here’s why:**
--You can get your card sooner because we can confirm your Veteran status more quickly.
--We can prefill part of your application based on your account details.
--You can save your form in progress, and come back later to finish filling it out.
4. Buttons: 
The last button should be Start the Veteran ID Card Application
as we don't use VIC acronym on this page. 
If we think that Start the Veteran ID Card Application will be too long, then we'll need to incorporate VIC into the copy on this page. 

signed in
1. We're missing the following bullet in the photo requirement list. It should go above that last bullet (Be in a .jpeg, .png, .bmp, or .tiff file format)
--Show what you look like now (a photo taken sometime in the last 10 years), **and**

2. Photo examples: 
please change header to: 
Examples of good photo IDs (to match how this title appears on the photo upload page)

3. bullet #2
This doesn't have some of the updated copy. The below still needs to be updated:
**In addition to providing extra security measures, when you’re signed in to your account, your application process can go more smoothly. Here’s why:**
--You can get your card sooner because we can confirm your Veteran status more quickly.
--We can prefill part of your application based on your account details.
--You can save your form in progress, and come back later to finish filling it out.

4. button
The last button should be Start the Veteran ID Card Application
as we don't use VIC acronym on this page. If we think that Start the Veteran ID Card Application will be too long, then we'll need to incorporate VIC into the copy on this page.

**Comment by jbalboni** [comment]
@peggygannon I made the updates and updated the screenshots.

**Review by peggygannon — APPROVED** [review]
@jbalboni LGTM! Thank you!

**Pull request merged**